framework/components/ecl-timelines/README.md

```diff
@@ -1,3 +1,14 @@
-# Timelines
+# Timeline and Time schedule
 
-The JS behaviour of this component needs to be rewritten without jQuery.
+## Why and how to use this component
+
+A timeline is a way of displaying a list of events in chronological order.
+
+There are two variants of this component:
+- **timeline**
+- **time schedule**
```

Comment:
Indeed, it's about the content, not the design, so there's no need to add anything for it.